### DISCUSSION TITLE
Best way to configure secondary 'completed' directories?

### DISCUSSION THREAD

**Discussion by philiprenich** [haugene/docker-transmission-openvpn#2856] [Q&A]
First off, I love how simple this Docker setup was to get running. (The only issue I had was forgetting to turn off the non-Dockerised VPN my server was already running 🙈) Many thanks to the maintainers!

I have mounted `/data` to a `/transmission` on my drive. As a sibling folders, unrelated to this Docker container, I have `/files` and `/stuff`. What's the best set up so that I can change the location of downloaded items from `/data/completed` (or rather `/transmission/completed` to `/files` or `/stuff`? I don't want to use `/:/data` as my volume. 

Initially, I thought I would symlink the sibling folders, but that ran in to permission issues that I wasn't quite sure how to solve.

What I am currently doing is adding extra mount points to my volumes with `/files:/data/files` and `/stuff:/data/stuff` That seems to work, though I'm a little concerned about the Docker container having full access to those (maybe that's unavoidable regardless of the setup?). 

Alongside this, files downloaded by Transmission are owned by root. Is there a way to set up the container so that the files are owned by my user on the host?

**Comment by pkishino** [comment]
Read the documentation, it explains how to set user/folder permissions/set folders etc

**Reply by philiprenich** [reply]
Gah, I did miss that user/group config setting. My bad! I'm a little unsure what the ramifications of altering `GLOBAL_APPLY_PERMISSIONS` are. What are the default set of permissions and what would they be w/out using this? Or is that in relation to configuring Transmissions umask settings?

Is my docker setup for volume mount points sensible though? Details around ways to do that I couldn't see in the docs.

Thanks (and sorry to be the guy who required a "rtfm" response 🤦🏽)

**Reply by rhythmrice** [reply]
> Read the documentation, it explains how to set user/folder permissions/set folders etc

where can i find that? ive spent hours trying to get exactly that setup

**Reply by philiprenich** [reply]
@rhythmrice Here's the config for users/groups: https://haugene.github.io/docker-transmission-openvpn/config-options/#user_configuration_options

In my docker-compose file I am mounting multiple volumes, the container's `/data` folder, `/config` folder and then a couple others `/data/x` and `/data/y` which map to where I want to easily be able to move. Now rather than a file going to `/data/completed` I can say `/data/x`. 

Additionally, I have the PUID and PGID set so that the transmission user on the host will own the files
